java/arcs/core/testutil/Fuzzing.kt

```diff
@@ -0,0 +1,344 @@
+package arcs.core.testutil
+
+import java.time.LocalDateTime
+import kotlin.random.Random
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.runBlocking
+
+/**
+ * This file contains utility classes and methods that establish fuzz-testing infrastructure.
+ *
+ * To write fuzz tests for a unit or assembly:
+ * (1) think of the invariants that make sense for the thing that you want to test.
```

Comment:
There's some style improvements you could make to this documentation. KDoc is basically markdown, so you can use all the usual markdown things to make it present nicely. Here's a few suggestions:

1. Use headings/subheadings to break it into smaller sections
2. Use numbered lists starting with `1.`, `2.`, etc.
3. Put code examples code snippets with three backticks (and designate the language as Kotlin via `'''kotlin` (using backticks not apostrophes))
4. Put inline references to real code symbols in square brackets
5. Put other inline code in backticks